lib/core/network/models/transaction_model.dart

```diff
@@ -1,72 +1,334 @@
 import 'package:equatable/equatable.dart';
 
-class TransactionModel extends Equatable {
+sealed class TransactionModel extends Equatable {
```

Comment:
@n13  i need u to review this class well. 

I am attempting to map Transactions to Typed TransactionModel in our models. Did i miss any transaction? If so, can you give me what the transaction looks like